syndeo-template/pom.xml

```diff
@@ -208,6 +208,7 @@
         <dependency>
             <groupId>org.apache.beam</groupId>
             <artifactId>beam-sdks-java-io-kafka</artifactId>
+            <version>${beam.snapshot.version}</version>
```

Comment:
nit: shouldn't use this if we are using the same version from the bom

Reply:
done